pkg/tss/Makefile

```diff
@@ -0,0 +1,6 @@
+IMAGE=tss
```

Comment:
this is great.  Let me know when you'd like me to set up the repo / signing, etc